app/views/welcome/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<h1> Welcome to Esty! </h1>
+<p> (not to be confused with Etsy) </p>
+
+<%= link_to "Admin", "/", data: {turbo_method: :get} %>
```

Comment:
I added this root file so we would have a landing page for our site and easier navigation to merchant/admin stuff